examples/src/main/python/ml/vector_slicer_example.py

```diff
@@ -20,9 +20,9 @@
 # $example on$
 from pyspark.ml.feature import VectorSlicer
 from pyspark.ml.linalg import Vectors
-from pyspark.sql.types import Row
 # $example off$
 from pyspark.sql import SparkSession
+from pyspark.sql.types import Row
```

Comment:
This probably should be be included in the example (they were before), so add an # $example on$ / # $example off$ around it.